packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -0,0 +1,67 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-skeleton {
+  border-radius: 2px;
+  background-color: rgba($gray1, 0.2);
+  width: 100%;
+  height: $pt-grid-size * 0.6;
+
+  // negative animation delay ensures animations are synced
```

Comment:
here's the SO thread that i found: http://stackoverflow.com/questions/15868431/keyframe-animations-not-in-sync

i'm not sure this actually solves the entire problem though. scrolling on BP table while showing skeletons will cause the skeletons to get out of sync. i suppose disabling scrolling is _okay_, but would be nice if we could enable scrolling if people want to preview how many rows of data there might be (if you know that kind of thing beforehand).